### PR TITLE
documents reference r4 enhancement

### PR DESCRIPTION
**Pull request by akashdeep677**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by akashdeep677** [comment]
Screenshot 2022-07-04 at 5 29 15 PM
Screenshot 2022-07-04 at 5 28 56 PM

File: content/millennium/r4/foundation/documents/document-reference.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ The following fields are returned if valued for clinical documents:
 
 ## Search
 
+Search operation is supported for clinical-note, cardiology, radiology, microbiology and pathology charted documents and clinical-note staged documents.
+
+Read operation is supported for clinical-note, cardiology, radiology, microbiology and pathology charted documents and clinical-note staged documents.
```
Comment (Sanvenky) [inline content/millennium/r4/foundation/documents/document-reference.md:43]: This should go under read implementation notes

File: content/millennium/r4/foundation/documents/document-reference.md [hunk 2]
```diff
@@ -58,12 +62,14 @@ Search for DocumentReferences that meet supplied query parameters:
...
  `_revinclude`            | N                  | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example: `_revinclude=Provenance:target`
+ `date`                   | N                  | [`date`]      | When this document reference was created
 
 _Implementation Notes_
 
 * When searching with the `period` parameter:
   * It must be provided twice, once with the `ge` prefix, and once with the `lt` prefix.
-  * If one `period` parameter includes a time, both must include a time.
+  * `Period` parameter  must include a time.
+  * `Period` and `date` search parameter cannot be provided together.
```
Comment (Sanvenky) [inline content/millennium/r4/foundation/documents/document-reference.md:72]: remove this line. Its covered down

**Review by Sanvenky — CHANGES_REQUESTED** [review]

**Comment by akashdeep677** [comment]
Please consider below PR -
https://github.com/cerner/fhir.cerner.com/pull/864

**Pull request closed without merging**